tests/fast-integration/kcp/client.js

```diff
@@ -81,13 +95,127 @@ class KCPWrapper {
         return await this.exec(args);
     }
 
+    async version() {
+        const args = [`--version`];
+        return await this.exec(args);
+    }
+
+    async upgradeKyma (subaccount, kymaUpgradeVersion) {
+        const args = [`--verbose=6`, `upgrade`, `kyma`, `--version`, `"${kymaUpgradeVersion}"`, `--target`, `subaccount=${subaccount}`];
```

Comment:
Will reduce verbosity later.

Reply:
I think the verbosity should be rather omitted and removed prior merging because it might print some sensitive information to the logs like access tokens, unless testing locally.